tensorboard/java/org/tensorflow/tensorboard/vulcanize/Vulcanize.java

```diff
@@ -593,6 +597,16 @@ private static boolean shouldIgnoreUri(String uri) {
         || (uri.contains("{{") && uri.contains("}}"));
   }
 
+  private static boolean isLibrarySourceName(String sourceName) {
+    final List<String> libraryPrefixes = ImmutableList.of("/iron-", "/neon-", "/paper-");
```

Comment:
Why not just inline these into a multiline if statement, with a bunch of || clauses? Then add a comment saying these are a bunch of Polymer standard library paths.